testsuite/features/step_definitions/docker_steps.rb

```diff
@@ -38,8 +38,8 @@ def retrieve_build_host_id
   )
 end
 
-Then(/^I am on the image page of "([^"]*)"$/) do |image_name|
-  step %(I navigate to "#{compute_image_url(image_name)}" page)
+Then(/^I am on the image store of the kiwi image for organization "([^"]*)"$/) do |org|
+  step %(I navigate to "os-images/#{org}/" page)
 end
```

Comment:
If we don't have a way from the WebUI to navigate into that URI, I suggest to create a card for Naica Squad to discuss a new entry in the menu for that. Ping @ncounter